### PR TITLE
Desktopversion PR

### PR DESCRIPTION
**Pull request by AlejandraKeber**
This part of the project includes the following:
- Modify the structure of the mobile version to be compatible with the responsive design and desktop design
- Add the HTML and CSS for the desktop version using media queries
- Add animations and transitions
- Use grid and flexbox to display elements on the page
- Replicate the design following the Figma Template 1
- Check and fix lintern errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,33 +1,49 @@
...
+            </nav>
+        </header>
+
         <main>
-            <section>
+            <section class="main">
                 <div>
                     <h1 class="headline">I'm Alejandra! <br>Glad to see you!</h1>
-                    <p class="headlinecont">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                    <p class="headlinecont">I’m a software developer! I can help you build a product , feature or
```
Comment (1GORDON) [inline index.html:30]: Great job @AlejandraKeber 🤝

- [ ] Kindly fix the background image of your main page. It appears to be overstretched which makes your design differ from the one provided in the Figma template.
- Your design
Your main

- Figma design
Figma main

File: index.html [hunk 2]
```diff
@@ -1,33 +1,49 @@
...
+                    <li><a href="#Portfolio">Portfolio</a></li>
+                    <li><a href="#About">About</a></li>
+                    <li><a href="#Contact">Contact</a></li>
+                </ul>
+            </nav>
+        </header>
+
         <main>
-            <section>
+            <section class="main">
```
Comment (1GORDON) [inline index.html:27]: Great job @AlejandraKeber 🤝

- [ ] You may also center the contents of your main page so as to reduce the big black gap left at the bottom part of this section.
Your main

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @AlejandraKeber ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
- [ ] Kindly remove the second author from your readme since you implemented this branch alone. You may also use [markdown syntax](https://www.markdownguide.org/basic-syntax/) to customize your readme file better so that we are directed to the correct Github profile for your Github handle in this template.
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ismailco — APPROVED** [review]
### Hi @AlejandraKeber, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**